src/test/java/hw2/ChromeTest.java

```diff
@@ -0,0 +1,46 @@
+package hw2;
+
+import org.openqa.selenium.WebDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+
+class ChromeTest {
+
+//    WebDriver driver;
+//
+//    @BeforeClass
+//    void setup() {
+//        WebDriverManager.chromedriver().setup();
+//        driver = new ChromeDriver();
+//        String url = "https://jdi-testing.github.io/jdi-light/index.html";
+//        driver.get(url);
+//    }
+//
+//    @AfterClass
+//    void teardown() {
+//        driver.quit();
+//    }
+
+    public static final String URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    WebDriver driver;
+
+    @BeforeClass
+    public void beforeClass(){
+        WebDriverManager.chromedriver().setup();
+        driver = WebDriverManager.chromedriver().create();
+    }
+
+    @BeforeMethod
+    public void setUp(){
+        driver.get(URL);
```

Comment:
Add next row to avoid any issues duo to small windows frame.
driver.manage().window().maximize();